.github/workflows/gradle_publish.yml

```diff
@@ -0,0 +1,32 @@
+name: Gradle Publish
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+    - name: Create local.properties with a read token
+      run: |
+        # Use a personal read token to install the Cyface Utils package
+        echo "github.user=${{ secrets.GITHUB_READ_ACCOUNT }}" > local.properties
+        echo "github.token=${{ secrets.GITHUB_READ_TOKEN }}" >> local.properties
+    - name: Build with Gradle
+      run: ./gradlew build
+    - name: Create local.properties with a repository token
+      run: |
+        # Use the repository's automatically set up token to pushlish to the registry
+        echo "github.user=cyface-de" > local.properties
+        echo "github.token=${{ secrets.GITHUB_TOKEN }}" >> local.properties
+    - name: Publish with Gradle
+      run: ./gradlew publishAll
```

Comment:
Please don't panic when looking into the [Package overview of this repository](https://github.com/cyface-de/android-backend/packages). I had a typo in the publish.gradle file earlier which published ~36 package variants with bad naming to the Github Registry. Unfortunately, the documentation did not warn about the fact that unce published, versions and packages cannot be deleted anymore by the owner (they removed this feature) - only when requested via the support. I did this already so the package overview page should be cleaned up soon. The current publish.gradle file only published the wanted packages:

- persistence (there is no movebis/cyface flavor here)
- synchronization
- synchronizationMovebis
- datacapturing
- datacapturingMovebis